modules/ai/assets/js/editor/helpers/focus-out-listener.js

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react';
+
+export function FocusOutListener( { children, listener, onFocusOut } ) {
+	const { indicatorRef, reset, remove, runAction } = listener;
+
+	React.useEffect( () => {
+		reset();
+
+		return () => {
+			remove();
+		};
+	}, [] );
+
+	return (
+		<>
+			{ children }
+
+			<input
+				style={ {
+					width: 0,
+					height: 0,
+					padding: 0,
+					margin: 0,
+					outline: 0,
+					border: 0,
+					opacity: 0,
+				} }
```

Comment:
```suggestion
					opacity: 0,
					position: 'fixed',
				} }
```
@DennisNerush  - to fix the vertical offset.